core/rwMRC.cpp

```diff
@@ -604,25 +627,31 @@ int ImageBase::writeMRC(size_t select_img, bool isStack, int mode, const String
 
     size_t imgEnd = (isStack)? Ndim : 1;
 
-    for ( size_t i = 0; i < imgEnd; i++ )
-    {
-        // If to also write the image data or jump its size
-        if (dataMode >= DATA)
+    if (checkMmapT(wDType) && !mmapOnWrite && dataMode >= DATA) {
+        writeData(fimg, 0, wDType, datasize_n * imgEnd, castMode);
+    } else {
+        for ( size_t i = 0; i < imgEnd; i++ )
         {
-            if (mmapOnWrite && Ndim == 1) // Can map one image at a time only
+            // If to also write the image data or jump its size
```

Comment:
?

Reply:
This is how the code looks now: 
![image](https://user-images.githubusercontent.com/26908093/96541269-f2f25580-129f-11eb-8a8d-032fd73160bb.png)


This is original version:  
![image](https://user-images.githubusercontent.com/26908093/96540601-6e530780-129e-11eb-9503-88103d0ddf81.png)

The only change that happened is highlighted by blue color. Github treats strangely the whole part of code from the screenshot.